terms-of-service-acceptance/verify/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Verify Terms of Service acceptance
+description: Verify Terms of Service acceptance
+
+inputs:
+  terms-of-service-acceptance-workflow-job-name:
```

Comment:
Sorry I asked the same question again below. I'm a bit surprised that this is a job name, since those are only unique in the context of a specific workflow.
I'll need to see a full working example to understand what's going on.

Reply:
Yes, I would have expected the check-runs endpoint to return a workflow name rather than a job name but this is what I experienced running it:
https://github.com/jprinet/test-gradle/actions/runs/6311401670/job/17135314077#step:2:42

with the corresponding workflow:
https://github.com/jprinet/test-gradle/blob/9f0f19616c088215873a3bb70d3415b5c3d9e5dc/.github/workflows/gradle-check-tos.yml#L9